tests/e2e/vsphere_volume_expansion.go

```diff
@@ -506,14 +548,12 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		ginkgo.By("File system resize should not succeed Since Vsan-health is down. Expect an error")
 		expectedErrMsg := "503 Service Unavailable"
 		framework.Logf("Expected failure message: %+q", expectedErrMsg)
-
 		err = waitForEvent(ctx, client, namespace, expectedErrMsg, pvclaim.Name)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 		ginkgo.By("Bringup vsanhealth service")
 		err = invokeVCenterServiceControl(startOperation, vsanhealthServiceName, vcAddress)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		isVsanhealthServiceStopped = false
```

Comment:
is this not required now?
Also, can you combine the AfterEach methods into one? Not sure why we need two.

Reply:
Added back isVsanhealthServiceStopped = false

New after each block introduced by latest merge. It was not there before.  Didn't notice this before